src/auth/screens/events.screen.js

```diff
@@ -149,7 +149,7 @@ class Events extends Component {
       case 'MemberEvent':
         return `${userEvent.payload.action}`;
       case 'PublicEvent':
-        return translate('auth.events.issuesEvent', language);
+        return translate('auth.events.publicEvent', language);
```

Comment:
It would be better to fix this in #193, because this refers to a localization error, although once there is a separate PR, can this be fixed here.